config/migrations/2024/20241126144504-rename-merged-municipalities-and-ocmws.sparql

```diff
@@ -0,0 +1,25 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?organisation skos:prefLabel ?oldName.
+  }
+} INSERT {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?organisation skos:prefLabel ?newName.
+  }
+} WHERE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?organisation a besluit:Bestuurseenheid;
+                  skos:prefLabel ?oldName.
+  }
```

Comment:
The `mock-login` person attached to these organizations need to be updated as well. We can add:
```sparql
GRAPH ?g {
  ?person foaf:member ?organisation ;
    foaf:familyName ?oldFamilyName .
}
```

`?g` is because they are in both the public graph and the organization graph.